impl/src/planning.rs

```diff
@@ -245,22 +251,22 @@ mod tests {
     assert!(krate_position.is_some());
 
     // Get crate context using computed position
-    let krate_context = crates_with_aliased_deps[krate_position.unwrap()].clone();
+    let krate_ctx = crates_with_aliased_deps[krate_position.unwrap()].clone();
```

Comment:
nit: any reason not to go with `crate_ctx` (understanding `crate` is not usable)?

Reply:
None, I was only mildly shortening the name that was already there.